org.eclipse.m2e.logback.feature/p2.inf

```diff
@@ -0,0 +1,22 @@
+#create a requirement on the fragment we are creating
+requires.0.namespace=org.eclipse.equinox.p2.iu
+requires.0.name=configure.logback.classis
+requires.0.range=[$version$,$version$]
+requires.0.greedy=true
+
+#create a IU fragment named configure.logback.classis
+units.0.id=configure.logback.classis
+units.0.version=$version$
+units.0.provides.1.namespace=org.eclipse.equinox.p2.iu
+units.0.provides.1.name=configure.logback.classis
+units.0.provides.1.version=$version$
+units.0.instructions.configure=setStartLevel(startLevel:2);\
+                               markStarted(started:true);
+units.0.hostRequirements.1.namespace=osgi.bundle
+units.0.hostRequirements.1.name=ch.qos.logback.classic
+units.0.hostRequirements.1.range=[1.3,1.5]
+units.0.hostRequirements.1.greedy=false
+units.0.requires.1.namespace=osgi.bundle
+units.0.requires.1.name=ch.qos.logback.classic
+units.0.requires.1.range=[1.3,1.5]
+units.0.requires.1.greedy=false
```

Comment:
@laeubi can you tell what I'm doing wrong here?
I want to set the `ch.qos.logback.classic` bundle as auto-started with level 2.

Reply:
If i rember correctly  you need to include the p2.inf into the build.properties.

Reply:
Yes that was definitely missing.
What was also missing was the `installBundle(bundle:${artifact})`. Otherwise logback.classic was just removed from the `bundles.info` file.